Documentation/getting-started.md

```diff
@@ -0,0 +1,49 @@
+# Getting started
+
+## Building the dex binary
+
+Dex requires a Go installation and a GOPATH configured. For setting up a Go workspace, refer to the [official documentation][go-setup]. Clone it down the correct place, and simply type `make` to compile the dex binary.
+
+```
+git clone https://github.com/coreos/dex.git $GOPATH/src/github.com/coreos/dex
+cd $GOPATH/src/github.com/coreos/dex
+make
+```
+
+A containerized version of this document using [rkt][rkt-repo] or Docker will be added later.
+
+## Configuration
+
+Dex is a single, scalable binary that pulls all configuration from a config file (no command line flags). Use the example config file defined in the `examples` folder to start up dex with an in-memory data store.
+
+```
+./bin/dex serve examples/config-dev.yaml
+```
+
+The [example config](../examples/config-dev.yaml) file documents many of the configuration options through inline comments. For extra config options, look at that file.
+
+## Running a client
+
+To interact with dex, like any other OAuth2 provider, users first visit a client app and are then redirected to dex to login. Dex ships with a example client app (also built with the `make` command), for testing and demos.
+
+The example app's flags default to user the statically configured OAuth2 client credentials in the `config-dev.yaml` file. Run the example app will cause it to query dex's [discovery endpoint][oidc-discovery] and determine the OAuth2 endpoints.
```

Comment:
Going to add this statement a little earlier.

> Use the [example config][example-config] file found in the `examples/` directory to start an instance of dex with an in-memory data store and a set of predefined OAuth2 clients.

Then change this to

> By default, the example client is configured with the same OAuth2 credentials defined in `examples/config-dev.yaml` to talk to dex. Running the example app will cause it to query dex's [discovery endpoint][oidc-discovery] and determine the OAuth2 endpoints.

That work?